pkg/sql/execinfrapb/processors_changefeeds.proto

```diff
@@ -60,6 +60,9 @@ message ChangeAggregatorSpec {
 
   // select is the "select clause" for predicate changefeed.
   optional Expression select = 6 [(gogoproto.nullable) = false];
+
+  // stmt is the entire create changefeed stmt.
+  optional string stmt = 7 [(gogoproto.nullable) = false];
```

Comment:
changes to protocol message usually require version gate; but more importantly... why do we need it? Wouldn't it better to simply retrieve job description from the job record itself (we have job id -- so presumably we can look up this information)